tensornetwork/tensor.py

```diff
@@ -111,7 +111,11 @@ def reshape(self, shape: Sequence[int]) -> "Tensor":
     """
     reshaped = self.backend.reshape(self.array, shape)
     return Tensor(reshaped, backend=self.backend)
-
+  def item(self):
+    """Return a Python scalar formed after converting a size-1 Tensor
```

Comment:
"Copies the single element of this `Tensor` and returns it. If this `Tensor` has more than one element,
raises `ValueError`."